beacon-chain/core/validators/validator.go

```diff
@@ -185,8 +185,8 @@ func ExitedValidatorIndices(state *pb.BeaconState) ([]uint64, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "could not get churn limit")
 	}
-	if uint64(exitQueueChurn) >= churn {
-		exitQueueEpoch++
+	if churn < uint64(exitQueueChurn) {
+		exitQueueEpoch = churn
```

Comment:
This was a bug in our previous impl - we should be clipping the exit queue epoch